packages/docs/getting-started/running-a-validator-in-mainnet.md

```diff
@@ -701,7 +701,7 @@ Next we will set up the Attestation Service itself. First, specify the following
 
 ```bash
 # On the Attestation machine
-export CELO_IMAGE_ATTESTATION=us.gcr.io/celo-testnet/celo-monorepo:attestation-service-rc1
+export CELO_IMAGE_ATTESTATION=us.gcr.io/celo-testnet/celo-monorepo:attestation-service-1-0-1
```

Comment:
at some point we should move off of the `celo-testnet` path

note also that the main celo image is still specified as:
```
export CELO_IMAGE=us.gcr.io/celo-testnet/celo-node:rc1
```

Reply:
Absolutely agreed. I believe we are moving more of release artifacts to docker hub proper soon